sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -555,4 +556,360 @@ public void beginRecognizeReceiptsWithOptions() throws IOException {
             });
         // END: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeReceipts#Flux-long-RecognizeReceiptsOptions
     }
+
+    /**
+     * Code snippet for {@link FormRecognizerAsyncClient#beginRecognizeBusinessCardsFromUrl(String)}
+     */
+    public void beginRecognizeBusinessCardsFromUrl() {
+        // BEGIN: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeBusinessCardsFromUrl#string
+        String businessCardUrl = "{file_source_url}";
+        // if training polling operation completed, retrieve the final result.
```

Comment:
not sure if this comment adds value here as it appears again in line 568 where you actually get the result

Reply:
should be a mistake